### PR TITLE
Code Review for [hello world back end] | Rails React integration with two projects

### PR DESCRIPTION
**Pull request by luigirazum**
# Hello Dear Code Reviewers Team 👋,
### The following requirements 📌 were fulfilled this time:
- [X] Create a new Rails API app called 'hello-rails-back-end'.
- [X] Initialize your project with Git.
- [X] Make sure that your project has a Postgres database set up. 
- [X] Create a table for storing your messages and create 5 different greetings. 
- [X] Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).

## ⚠️ Important Note
- This is the Back End project which interacts with the Front End available in the [https://github.com/luigirazum/hello-react-front-end](https://github.com/luigirazum/hello-react-front-end/)

- A Pull request with changes is available at the following link [Front End Pull Request](https://github.com/luigirazum/hello-react-front-end/pull/1).

### Luis 🙋‍♂️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,262 @@
...
+  **Hello Rails React** is a sample project that shows how to integrate Rails and React.
+  It is composed of two projects, one for the back end and another one to for the front end.
+  
+  ### Ruby on Rails
+  - RoR is used for the back end, and it is configured as an api, so it attends the requests of the front end.
+  - [`hello-rails-back-end`](https://github.com/luigirazum/hello-rails-back-end) this repository, is the one that holds the RoR project.
+
+  ### React
+  - React is used for the front end, and it consumes the back end as an API.
+  - [`hello-rails-front-end`](https://github.com/luigirazum/hello-rails-front-end), is the repository that holds the React project.
```
Comment (OyePriscilla) [inline README.md]: - [ ]  Kindly fix this broken link to in order to make your README file more professional. I got 404 error. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @luigirazum 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Implement a connection between a Ruby on Rails back-end and React front-end. ✔️ 

## Required Changes ♻️

**Kindly check the comment below**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (luigirazum) [reply README.md]: Thanks for you feedback @OyePriscilla.
The required changes were applied within this [commit](https://github.com/luigirazum/hello-rails-back-end/pull/1/commits/7fe16cc8a394125c09213a2eaf4100ebae921003)

**Review by Reem-lab — APPROVED** [review]
# Hi @luigirazum,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ]  Descriptive PR⭐
- [ ]  No Linters error 🤩
- [ ] Great job working on the API 💪🏻
- [ ] Good job working on the controller, model and routes ⭐🙏🏻



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**